configs/components/windows_puppet.json

```diff
@@ -1,4 +0,0 @@
-{
-  "url": "git://github.delivery.puppetlabs.net/puppetlabs-puppet_for_the_win.git",
-  "ref": "origin/master"
-}
```

Comment:
Yes, 👍  - that should also be removed.

But not only that.  I think everything from `bin` can be removed as well, except for `install-chocoately.ps1` which is being used in the config definitions.

So remove:

* build-cpp-pcp-client.ps1
* build-facter.ps1
* build-helpers.ps1
* build-leatherman.ps1
* build-pxp-agent.ps1
* build-windows.rb
* windows-env.ps1
* windows-toolset.ps1